keyboards/iriskeyboards/iriskeyboards.json

```diff
@@ -0,0 +1,239 @@
+{
```

Comment:
This file goes in the VIA repo, not here.

Reply:
Sorry about that! 

Thank you for what you guys do, QMK is fantastic!